README.md

```diff
@@ -25,3 +25,81 @@ Synopsis:
 ---------
 
 See the docstrings in `textgrid.py`
+
+Example:
+---------
+This is a simple example of reading a TextGrid file. 
+```
+import textgrid
+
+# Read TextGrid object from file
+tg = textgrid.TextGrid.fromFile('test.TextGrid')
+
+# Interval example
+print ("------- Interval example -------")
```

Comment:
Please use your best Python style---this is supposed to be a readable example for newbies, right?

* Please don't use space between "print" and the parentheses, I didn't even know the Python grammar allowed that and it's very distracting.
* Comments should have the syntax of a sentence and be punctuated as such.